docs/topics/coroutines-guide.md

```diff
@@ -32,6 +32,6 @@ In order to use coroutines as well as follow the examples in this guide, you nee
 
 ## Additional references
 
-* [Guide to UI programming with coroutines](../../ui/coroutines-guide-ui.md)
+* [Guide to UI programming with coroutines](https://github.com/Kotlin/kotlinx.coroutines/blob/master/ui/README.md)
```

Comment:
That's wrong link, too. It shall point here: https://github.com/Kotlin/kotlinx.coroutines/blob/master/ui/coroutines-guide-ui.md